server/services/v1/api.go

```diff
@@ -18,6 +18,9 @@ import (
 	"context"
 	"net/http"
 
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/tigrisdata/tigris/server/metrics"
+
```

Comment:
is this added by goimports? Try removing all extra lines and then run goimports again.